app/assets/javascripts/modules/nav_off_canvas.js.coffee

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) 2018, hitobito AG. This file is part of
+#  hitobito_dsj and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+app = window.App ||= {}
+
+app.NavOffCanvas = {
+  toggle: (e) ->
+    e.preventDefault();
+    $('.nav-left').toggleClass("is-visible");
+}
+
+$(document).on('click', '.toggle-nav', app.NavOffCanvas.toggle)
+$(document).on('click', '.nav-left-overlay', app.NavOffCanvas.toggle)
+$(document).keyup (e) ->
+  if (e.key == 'Escape') && $('.nav-left').hasClass('is-visible')
+    e.preventDefault;
```

Comment:
Gibt es einen Grund weshalb du hier nicht deine eigene toggle Methode verwendest?
& Bitte noch all Semikolons entfernen, die werden in coffee-script nicht benötigt